### DISCUSSION TITLE
Opening option for sponsoring

### DISCUSSION THREAD

**Discussion by gabriellemon** [encoredev/encore#1395] [Suggestions]
Hey. I self host encore but don't use the pro option due to using big-cloud. I would like to sponsor the project. Thanks.

**Comment by luisnquin** [comment]
Are you going to go ahead with this? I'd like to contribute too

**Comment by marcuskohlberg** [comment]
Right now we're not intending to open up sponsoring as "the juice isn't worth the squeeze" (we would need to establish accounting processes for this etc.).

But we REALLY appreciate your interest in support the project!
And if you want to do something right now to support Encore, the biggest help is probably to blog/post/tweet/tell a friend about it. With every new user we get stronger!
